src/static/static/img/cc_icon.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg">
+  <!-- DO NOT MOVE OR RENAME THIS FILE OR HISTORICAL USAGES OF THE ATTRIBUTION HTML GENERATED BY THE APP WILL BREAK -->
```

Comment:
Should we add the same comment to the other icons or in a README in this folder?

Reply:
Oh true! Maybe I'll just create a "DO_NOT_MOVE_THIS_DIRECTORY.md" with an explanation and we can somehow exclude it from Nuxt's static file serving so it doesn't end up getting crawled by some very curious crawler or something :joy:

Reply:
Turns out Nuxt will ignore hidden files in the static folder :+1: 

Added a very loud README in https://github.com/WordPress/openverse-frontend/pull/990/commits/2c5b05e65746a414427976f519ec61e1f12495ca